en/backup-by-ebs-snapshot-across-multiple-kubernetes.md

```diff
@@ -0,0 +1,282 @@
+---
+title: Back Up a TiDB Cluster across multiple Kubernetes Using EBS Volume Snapshots
+summary: Learn how to back up TiDB cluster data across multiple Kubernetes to S3 based on EBS volume snapshots using BR Federation Manager.
+aliases: ['/docs/tidb-in-kubernetes/dev/backup-by-ebs-snapshot-across-multiple-kubernetes/']
+---
+
+# Back Up a TiDB Cluster across multiple Kubernetes Using EBS Volume Snapshots
+
+This document describes how to back up the data of a TiDB cluster across multiple AWS Kubernetes to AWS storage based on EBS volume snapshots.
+
+The backup method described in this document is implemented based on CustomResourceDefinition (CRD) in BR Federation Manager and TiDB Operator. For the underlying implementation, [BR](https://docs.pingcap.com/tidb/stable/backup-and-restore-overview) is used to get the backup data of the TiDB cluster, and then send the data to the AWS storage. BR stands for Backup & Restore, which is a command-line tool for distributed backup and recovery of the TiDB cluster data.
```

Comment:
What is BR Federation Manager? Can you explain it or link to somewhere that explains it?

Reply:
BR Federation Manager is an operator in control plane, like tidb operator, but it can access multiple Kubernetes and only uses for backup and restore. It orchestrate `Backup` and `Restore` CR across multiple Kubernetes to complete the backup and restore. I will add a doc about its architecture and link to it later.